src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,70 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FC } from 'react';
+import { SortType } from '../../enums/SortType';
+import { getTodoType } from '../../helpers/getTodoType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string;
+  onQueryChange: (query: string) => void;
+  sortType:SortType,
+  onSortTypeChange: (type: SortType) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: FC<Props> = ({
+  query,
+  onQueryChange,
+  sortType,
+  onSortTypeChange,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={sortType}
+            onChange={
+              (event) => onSortTypeChange(getTodoType(event.target.value))
```

Comment:
It's better to create a handler so that a function is not created for every render.